### PR TITLE
Update Tagging

### PR DESCRIPTION
**Pull request by PanSpagetka**
Update Tagging according to https://github.com/patternfly/patternfly-react/pull/351

Update reducers, tests, stories to be consistent with these changes. 

Because it looks PF PR may not be merged soon, introduce back css files.

### CODE REVIEW

File: src/tagging/components/Tagging/Tagging.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import CategoryModifier from '../InnerComponents/CategoryModifier';
+import ValueModifier from '../InnerComponents/ValueModifier';
+
+class Tagging extends React.Component {
+  onTagValueChange = selectedTagValue => {
+    const action = {
+      tagCategory: this.props.selectedTagCategory,
+      tagValue: selectedTagValue
+    };
+    if (this.findSelectedCat(this.props.selectedTagCategory).singleValue) {
```
Comment (Hyperkid123) [inline src/tagging/components/Tagging/Tagging.js]: typo? maybe `findSelectedTag`? instead of cat

File: src/tagging/components/Tagging/Tagging.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  onTagDeleteClick = (tagCategory, tagValue) => {
+    this.props.onTagDeleteClick({ tagCategory, tagValue });
+  };
+
+  getTagValues = () =>
+    (this.findSelectedCat(this.props.selectedTagCategory) &&
+      this.findSelectedCat(this.props.selectedTagCategory).values) ||
+    [];
+
+  findSelectedCat = (selectedTagCategory = { id: undefined }) =>
```
Comment (Hyperkid123) [inline src/tagging/components/Tagging/Tagging.js]: `tag` not `cat`

File: src/tagging/tests/category.modifier.test.js [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
 ];
 
 describe('TagCategory Component', () => {
   it('match snapshot', () => {
-    const component = renderer.create(<CategoryModifier
-      selectedTagCategory={selectedTagCategory}
-      onTagCategoryChange={onTagCategoryChange}
-      tagCategories={tagCategories}
-    />);
+    const component = renderer.create(
```
Comment (Hyperkid123) [inline src/tagging/tests/category.modifier.test.js]: Please change this to enyzme, reactRenderer will be removed in future. Examples in #39

File: src/tagging/tests/tagging.test.js [hunk 4]
```diff
@@ -1,25 +1,60 @@
...
+const selectedTagValue = { description: 'duck', id: 1 };
+const assignedTags = [
+  {
+    description: 'Name',
+    id: 1,
+    tagValues: [{ description: 'Pepa', id: 11 }]
+  }
+];
 
 function onChange(x) {
```
Comment (Hyperkid123) [inline src/tagging/tests/tagging.test.js]: Use `jest.fn()` for mock functions

File: src/tagging/components/InnerComponents/TagView.js [hunk 5]
```diff
@@ -36,24 +36,20 @@ class TagView extends React.Component {
...
+    }).isRequired).isRequired,
+  })).isRequired,
   onTagDeleteClick: PropTypes.func.isRequired,
-  header: PropTypes.string
+  header: PropTypes.string,
 };
 
 TagView.defaultProps = {
-  header: 'Assigned tags'
+  header: 'Assigned tags',
```
Comment (Hyperkid123) [inline src/tagging/components/InnerComponents/TagView.js]: i18n

**Comment by Hyperkid123** [comment]
Seems OK

File: src/tagging/tests/tagging.test.js [hunk 6]
```diff
@@ -1,62 +1,104 @@
...
+        selectedTagCategory={selectedTagCategory}
+        selectedTagValue={selectedTagValue}
+      />
+    );
     wrapper.instance().onTagCategoryChange('xaxa');
-    expect(onTagCategoryChange.mock.calls.length).toEqual(1);
+    expect(onTagCategoryChange.mock.calls).toHaveLength(1);
     wrapper.instance().onTagValueChange('wawa');
-    expect(onTagCategoryChange.mock.calls.length).toEqual(1);
+    expect(onTagCategoryChange.mock.calls).toHaveLength(1);
```
Comment (himdel) [inline src/tagging/tests/tagging.test.js]: @PanSpagetka Are you testing that `onTagCategoryChange` does *not* get called when you call `onTagValueChange`?

Or is this a copypaste bug and the other 2 functions should also have 1 call?

Comment (PanSpagetka) [reply src/tagging/tests/tagging.test.js]: Such PR necromancy :D I don't really remember, but I think that it is testing that each function was called once. I am sorry, but I don't understand where is the problem?

Comment (himdel) [reply src/tagging/tests/tagging.test.js]: Your calling each of the 3 functions...

and then testing the first one was called only once, and testing it 3 times ;)

**Pull request merged**